mypy/fastparse.py

```diff
@@ -487,11 +487,12 @@ def translate_stmt_list(
             and self.type_ignores
             and min(self.type_ignores) < self.get_lineno(stmts[0])
         ):
-            if self.type_ignores[min(self.type_ignores)]:
```

Comment:
We can't use walrus just yet; we still support 3.7

Reply:
This is the first time I've used walrus in real code :)
Probably, the last time.